ntc_templates/templates/edgecore_show_interfaces_status.textfsm

```diff
@@ -0,0 +1,75 @@
+Value Required INTERFACE (.*)
+Value PORT_TYPE (.*)
+Value ADDRESS (.*)
+Value NAME (.*)
+Value PORT_ADMIN (.*)
+Value MDIX_MODE (.*)
+Value SPEED_DUPLEX (.*)
+Value CAPABILITIES (.*)
+Value BROADCAST_STORM (.*)
+Value BROADCAST_STORM_LIMIT (.*)
+Value MULTICAST_STORM (.*)
+Value MULTICAST_STORM_LIMIT (.*)
+Value UNKNOWNUNICAST_STORM (.*)
+Value UNKNOWNUNICAST_STORM_LIMIT (.*)
+Value FLOW_CONTROL (.*)
+Value VLAN_TRUNKING (.*)
+Value LACP (.*)
+Value PORT_SECURITY (.*)
+Value MAX_MAC_COUNT (.*)
+Value PORT_SECURITY_ACTION (.*)
+Value MEDIA_TYPE (.*)
+Value GIGA_PHY_MODE (.*)
+Value LINK_STATUS (.*)
+Value OPERATION_STATUS (.*)
+Value OPERATION_SPEED_DUPLEX (.*)
```

Comment:
Break into multiples

Reply:
Splited it to the 2 new values:
 - speed
 - duplex